WordPress/Classes/ViewRelated/Reader/WPTableViewHandler.m

```diff
@@ -522,6 +522,7 @@ - (void)controller:(NSFetchedResultsController *)controller
         {
             [self invalidateCachedRowHeightAtIndexPath:indexPath];
             [self.tableView reloadRowsAtIndexPaths:@[indexPath] withRowAnimation:self.updateRowAnimation];
+            self.indexPathSelectedAfterUpdates = self.tableView.indexPathForSelectedRow;
```

Comment:
I've caught a couple scenarios in which an entity might get refreshed multiple times, and the selected row was getting lost.

You mean, it might conflict with the logic in `NSFetchedResultsChangeMove`, if we get multiple changes in batch right?. (Which is definitely a scenario i missed in my tests!!!!!)